src/ucp/proto/proto_am.inl

```diff
@@ -23,6 +23,29 @@
 
 typedef void (*ucp_req_complete_func_t)(ucp_request_t *req, ucs_status_t status);
 
+static UCS_F_ALWAYS_INLINE ucs_status_t ucp_am_handle_user_header_send_status(
+        ucp_request_t *req, ucs_status_t send_status)
+{
+    ucs_status_t status;
+
+    if (ucs_unlikely((send_status == UCS_ERR_NO_RESOURCE) &&
```

Comment:
can do the unlikely only on `send_status == UCS_ERR_NO_RESOURCE` part and not the whole expression